### ISSUE TITLE
Depedency Resolution Issue

### ISSUE THREAD

**Issue by maleghast** [Complexible/stardog-clj#4]
Hi,

I have a working / running Stardog Community install, and so I created a basic Clojure App and then added the dependency for stardog-clj to my project.clj, and then ran lein deps...

I get the following output at the end of the run:

```Could not find artifact org.openrdf.sesame:sesame:jar:2.7.14 in central (https://repo1.maven.org/maven2/)
Could not find artifact org.openrdf.sesame:sesame:jar:2.7.14 in clojars (https://clojars.org/repo/)
Could not find artifact org.openrdf.sesame:sesame:jar:2.7.14 in stardog (http://maven.stardog.com)
This could be due to a typo in :dependencies or network issues.
If you are behind a proxy, try setting the 'http_proxy' environment variable.```

I am not behind an http_proxy, and having visited the repositories mentioned in the error I note that it is present at https://repo1.mave.org/maven2.

I have installed the jar into my local maven repository manually and yet still Leiningen will not resolve the dependency or allow me to run my code.

I admit to being essentially ignorant of Maven - I am not a Java programmer per se, just a Clojure programmer and I accept that I rely on Leiningen far too heavily, but these facts simply amplify my need for assistance. :-(

Help?

**Comment by AlBaker** [comment]
Hi, 

You are correct, there is an issue in the dependencies.  If you had a previous version of Stardog installed that ran the `mavenInstall.sh`, then it would mask the issue.  The fix is in this commit 835f8debcbbc72813890fb82d0200cd12518989d

Clojars won't let you publish over an existing artifact, so I'll release a 3.0.1 and 3.0.2 with the corrected dependencies.

**Comment by AlBaker** [comment]
This fix is now available in Clojars with stardog-clj 3.0.1 and 3.0.2 releases
